#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_dash_workaround.dpatch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Work around dash's bug #337294 so that we can build fine when sh is dash
## DP: bz#314961

@DPATCH@

=== configure.in
==================================================================
--- xulrunner/configure.in	(revision 22)
+++ xulrunner/configure.in	(local)
@@ -5246,7 +5246,7 @@
 [ if test "$enableval" != "no"; then
     MOZ_OPTIMIZE=1
     if test -n "$enableval" && test "$enableval" != "yes"; then
-        MOZ_OPTIMIZE_FLAGS=`echo $enableval | sed -e 's|\\\ | |g'`
+        MOZ_OPTIMIZE_FLAGS=`echo "$enableval" | sed -e 's|\\\ | |g'`
         MOZ_OPTIMIZE=2
     fi
 else
